### PR TITLE
Python Support for Functional Graphs (Silver)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by ryanchou-dev** [comment]
resolves #3703

File: content/3_Silver/Func_Graphs.mdx [hunk 1]
```diff
@@ -102,6 +102,53 @@ $2c\left\lceil \frac{t}{c}\right\rceil$ queries. Then we can find the first node
...
+
+while 1:  # t times
+	qry([i for i in range(10)])
+	process()
+
+	if pos[0] == pos[2]:
+		break
+
+print("done")
+sys.stdout.flush()
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:149]: what's w/ the flulsh

File: content/3_Silver/Func_Graphs.mdx [hunk 2]
```diff
@@ -102,6 +102,53 @@ $2c\left\lceil \frac{t}{c}\right\rceil$ queries. Then we can find the first node
...
+while 1:  # c * ceil(t / c) times
+	qry([0, 1])
+	process()
+	qry([0])
+	process()
+
+	if pos[0] == pos[1]:
+		break
+
+while 1:  # t times
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:141]:
```suggestion
while True:  # t times
```

File: content/3_Silver/Func_Graphs.mdx [hunk 3]
```diff
@@ -102,6 +102,53 @@ $2c\left\lceil \frac{t}{c}\right\rceil$ queries. Then we can find the first node
...
+	global pos
+
+	s = input().split()
+
+	for j in range(int(s[0])):
+		for k in s[j + 1]:
+			pos[int(k)] = j
+
+
+while 1:  # c * ceil(t / c) times
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:132]: same here

File: content/3_Silver/Func_Graphs.mdx [hunk 4]
```diff
@@ -665,6 +802,38 @@ public class badge2 {
...
 
+<PySection>
+
+```py
+import sys
+
+# may need for large inputs
+sys.setrecursionlimit(1000000)
+
+vis = [0] * MAXN
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:813]: i don't see maxn defined anywhere

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
thanks

File: content/3_Silver/Func_Graphs.mdx (hunk 4 above)
Comment (ryanchou-dev) [reply content/3_Silver/Func_Graphs.mdx:813]: yeah it's sample code -- the c++ solution does something similar

File: content/3_Silver/Func_Graphs.mdx [hunk 5]
```diff
@@ -102,6 +102,52 @@ $2c\left\lceil \frac{t}{c}\right\rceil$ queries. Then we can find the first node
...
+while True:  # c * ceil(t / c) times
+	qry([0, 1])
+	process()
+	qry([0])
+	process()
+
+	if pos[0] == pos[1]:
+		break
+
+while True:  # t times
```
Comment (envyaims) [inline content/3_Silver/Func_Graphs.mdx:141]:
```suggestion
while True:  # at most t times
```

**Review by envyaims — APPROVED** [review]
hug

File: content/3_Silver/Func_Graphs.mdx (hunk 4 above)
Comment (SansPapyrus683) [reply content/3_Silver/Func_Graphs.mdx:812]: that seems like an issue
i'll remind myself to do a maintenance pr sometime

**Comment by ryanchou-dev** [comment]
yes! im trying to keep the code consistent between languages...

**Pull request merged**